Kickstarter-iOS/Features/PledgePaymentMethods/Views/Cells/PostCampaignPledgeRewardsSummaryCell.swift

```diff
@@ -119,7 +119,7 @@ private func headerLabelStyle(_ label: UILabel) {
 
 private func labelStyle(_ label: UILabel) {
   label.font = UIFont.ksr_subhead().bolded
-  label.textColor = UIColor.ksr_support_400
+  label.textColor = label.text == Strings.Bonus_support() ? UIColor.ksr_black : UIColor.ksr_support_400
```

Comment:
Nice catch! I did double check this in the figma, and there's a bunch of other little things we could be doing in this table, including adding a per-item cost when there's multiple of an add-on, which I literally just noticed was in there.